ts-tests/tests/utils.ts

```diff
@@ -67,7 +67,7 @@ export async function initApiPromise(config: any): Promise<ParachainConfig> {
     // Set Eve's balance to 1000000000000000
     const eve_info = await api.query.system.account(eve.address);
     if (eve_info.data.free.lt(new BN(1000000000000000))) {
-        const txSetBalance = api.tx.sudo.sudo(api.tx.balances.setBalance(eve.address, 1000000000000000, 0));
+        const txSetBalance = await sudoWrapper(api, api.tx.balances.setBalance(eve.address, 1000000000000000, 0))
```

Comment:
I guess this won't work, right?
If we don't need the `if` branch, we can simply delete it.

Reply:
i think we should keep it, if delete it, the test will fail in litentry/rococo